src/lib/TranscriptEditor/Settings/TimecodeOffset/index.js

```diff
@@ -23,6 +23,13 @@ class TimecodeOffset extends React.Component {
   resetTimecodeOffset = () => {
     const resetTimecodeOffsetValue = 0;
 
+    this.props.handleAnalyticsEvents({ 
+      category: 'Settings - TimecodeOffset', 
```

Comment:
for category I am just using the name of the component. But in case of the `TimecodeOffset`, I specified the parent component otherwise it might be harder to remember where that comes from  etc.. but ok point taken changing it to `TimecodeOffset` to stick to the convention